ovs/lib/storagedriver.py

```diff
@@ -178,7 +179,7 @@ def manual_voldrv_arakoon_checkup():
         StorageDriverController._voldrv_arakoon_checkup(True)
```

Comment:
We execute this `manual_voldrv_arakoon_checkup` in the `add_vpool`, so it might be discarded due to a running task and we don't wait for it. We either handle this manually (return `True` here, and check for that value in `add_vpool`), or adapt `ensure_single` to support waiting in default mode.